resources/views/includes/structure/nav.blade.php

```diff
@@ -47,8 +47,30 @@
 
 
 <!-- Nav bars -->
+
+@if(
+(!empty(SiteHelper::getAnnouncement()['data'][0]['status']) && SiteHelper::getAnnouncement()['data'][0]['status'] == 'published')
```

Comment:
It would be useful to add a comment in here explaining what the code is for, e.g. 

```
{{-- If the client has a global announcement published in the CMS, then output it at the top of the page --}}
```